### PR TITLE
INT-3683: Fix typo in the `schema.sql.vpp`

### PR DESCRIPTION
**Pull request by artembilan**
JIRA: https://jira.spring.io/browse/INT-3683
- In addition fix `inputs` for the Gradle `generateSql` task and regenerate all SQLs
  to apply the typo fix
- Applying the Travis config fix to this commit as well.

### CODE REVIEW

**Comment by garyrussell** [comment]
Do you think this is worth it? Seems unfortunate to make a breaking change for a minor typo in a PK name.

**Comment by artembilan** [comment]
> Do you think this is worth it?

Yes, it is worth do not make us untidy. 
From one side we always can add Migration Guice not.
From other side see my comment in JIRA: good teams always have good DBA, who won't apply third-party scripts to his DB without the proper review.
So, I insist on this fix.

File: spring-integration-jdbc/build.gradle [hunk 1]
```diff
@@ -45,7 +47,3 @@ task generateSql {
 
 // tie schema generation to the build lifecycle
 compileJava.dependsOn generateSql
-
-task cleanSql (type: Delete) {
```
Comment (garyrussell) [inline spring-integration-jdbc/build.gradle]: Can you explain these changes in more detail (in the commit comment).

**Comment by garyrussell** [comment]
Otherwise, LGTM

File: spring-integration-jdbc/build.gradle (hunk 1 above)
Comment (artembilan) [inline spring-integration-jdbc/build.gradle]: Pushed. See commit comment.
I'd be glad to hear from you that it works properly not only from my side.
I mean this to check should be enough:
```
./gradlew clean :spring-integration-jdbc:testAll
```

**Comment by artembilan** [comment]
Pushed the fix for HTTP test, which fails on CI with `--parallel` for Gradle

**Comment by artembilan** [comment]
Pushed increased poll timeout for the `ProducerAndConsumerAutoStartupTests`

**Comment by artembilan** [comment]
Pushed `DelayerUsageTests` fix

**Comment by artembilan** [comment]
@garyrussell , let me know if it's OK what I am doing here with those fixes for issues from CI.
Maybe you'd prefer to have them in the separate PR...

**Comment by garyrussell** [comment]
This is fine.

**Comment by artembilan** [comment]
Pushed generics warning fix according to the Travis advice

**Comment by artembilan** [comment]
Pushed fix for WebSocket test.
If that works, there will be need to apply the same `response timeout` for all Tomcat instances in our tests.

**Comment by artembilan** [comment]
Repushed after rebasing and fixing for conflicted

**Comment by artembilan** [comment]
Pushed the fix for the last Travis' barking.

**Comment by garyrussell** [comment]
````
java.lang.IllegalStateException: java.lang.ClassCastException: java.lang.Long cannot be cast to java.lang.String
	at org.springframework.integration.websocket.ClientWebSocketContainer.getSession(ClientWebSocketContainer.java:118)
	at org.springframework.integration.websocket.ClientWebSocketContainerTests.testClientWebSocketContainer(ClientWebSocketContainerTests.java:81)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)
Caused by: java.lang.ClassCastException: java.lang.Long cannot be cast to java.lang.String
	at org.apache.tomcat.websocket.WsWebSocketContainer.connectToServer(WsWebSocketContainer.java:276)
	at org.springframework.web.socket.client.standard.StandardWebSocketClient$1.call(StandardWebSocketClient.java:152)
	at org.springframework.web.socket.client.standard.StandardWebSocketClient$1.call(StandardWebSocketClient.java:149)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:745)

````

**Comment by artembilan** [comment]
Pushed.
Doh... I had to test at least that changed test-case...

**Comment by artembilan** [comment]
@garyrussell , isn't it a time to take a look to the IP tests one more time?
The current Travis result:
```
org.springframework.integration.ip.tcp.connection.FailoverClientConnectionFactoryTests > testRealNetSingleUse FAILED
    org.springframework.messaging.MessagingException at FailoverClientConnectionFactoryTests.java:377
        Caused by: java.net.SocketException
```
Something similar I saw today on CI.

**Comment by garyrussell** [comment]
I'll get to it (IP) one day :frowning:

**Comment by garyrussell** [comment]
Merged as 178c86faa4e5e43d963c45ab5ab754195da6826c

**Pull request closed without merging**